dashproducts.php

```diff
@@ -197,7 +197,7 @@ public function getTableBestSellers($date_from, $date_to)
 					SELECT
 						product_id,
 						product_name,
-						SUM(product_quantity) as total,
+						SUM(product_quantity-product_quantity_refunded-product_quantity_return-product_quantity_reinjected) as total,
```

Comment:
Did you verify a product can't be refunded, returned and reinjected as the same time in the database?

We would have negative sums if this is true.

Reply:
there's no way that the sum of product_quantity_refunded, product_quantity_return and product_quantity_reinjected be superior then product_quantity for a given product in an Order, unless you did it manually in DB.
And, when I tried it, it will not display neither the list nor error message.